app/assets/stylesheets/application.scss

```diff
@@ -19,6 +19,13 @@
  * Include any third party libraries or frameworks here.
  */
 
+
+$brand-primary:         darken(#428bca, 6.5%) !default; // #337ab7
+$brand-success:         #5cb85c !default;
+$brand-info:            #5bc0de !default;
+$brand-warning:         #f0ad4e !default;
+$brand-danger:          #d9534f !default;
```

Comment:
@tillprochaska we're referencing these variables in `'components/markup'`. Can we live without 'em?